src/main/java/redis/clients/jedis/search/aggr/AggregationBuilder.java

```diff
@@ -149,4 +149,36 @@ public String getArgsString() {
   public boolean isWithCursor() {
     return isWithCursor;
   }
+
+  public AggregationBuilder verbatim() {
+    args.add(1, "VERBATIM");
```

Comment:
Let the user call in proper order.
```suggestion
    args.add("VERBATIM");
```

Reply:
Updated it. But just curious on why we don't want the library to handle it ?
Because we know that this argument should be the first for this command if it is used.